checkov/cloudformation/graph_builder/local_graph.py

```diff
@@ -335,6 +339,22 @@ def _create_edge(self, origin_vertex_index: int, dest_vertex_index: int, label:
             self.out_edges[origin_vertex_index].append(edge)
             self.in_edges[dest_vertex_index].append(edge)
 
+
+    def calculate_encryption_attribute(self) -> None:
+        for vertex_index in self.vertices_by_block_type.get(BlockType.RESOURCE, []):
+            vertex = self.vertices[vertex_index]
+            resource_type =  vertex.id.split(".")[0]
```

Comment:
If I'm not mistaken we could fetch the resource_type from `vertex.attributes.resource_type`, it would be a more stable approach, because if I recall correctly, the resource type might not always be in `vertex.id.split(".")[0]`

Reply:
@Saarett  the problem with that is vertex.attributes.get('resource_type') is a list. so im not sure how can i use it here